Learning-Tools-Interoperability.md

```diff
@@ -0,0 +1,85 @@
+# Learning Tools Interoperability (LTI)
+
+MarkUs integrates with other Learning Management Systems (LMS) via the LTI standard.
+Currently MarkUs supports the following LMS platforms:
+
+- Canvas
+
+## For LMS Administrators
+
+### Installing MarkUs on an LMS
+
+Each LMS implements their own LTI integration process.
+Typically, only administrators can add LTI integrations.
+
+### Canvas
+
+To add MarkUs to a Canvas instance, see their page on
+[configuring LTI keys](https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-LTI-key-for-an-account/ta-p/140).
+
+MarkUs can be added via a configuration URL. The configuration URL is available at  /lti_deployments/get_canvas_config
```

Comment:
You should elaborate this section by referring to which specific part of the Canvas documentation is relevant. For example, a search for the phrase "configuration URL" on the Canvas documentation doesn't yield any hits.